src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/Court.kt

```diff
@@ -39,7 +39,7 @@ class Court(
 
     other as Court
 
-    return courtId == other.courtId
+    return courtId == other.courtId && code == other.code
```

Comment:
Technically id is enough for equality check.

Adding in the code makes it easier for unit testing as you cannot provide and ID in tests as the field is private.

There is are side effects as a result of this change.